soc/nordic/common/dmm.h

```diff
@@ -23,19 +23,27 @@ extern "C" {
 
 /** @cond INTERNAL_HIDDEN */
 
+#ifdef CONFIG_DCACHE
+
 /* Determine if memory region is cacheable. */
-#define DMM_IS_REG_CACHEABLE(node_id)					 \
```

Comment:
Was there any issues with that macro?

Reply:
It was suggested by @anangl. Not interested in going back and forth on this, it's an urgent fix.